charts/k8ssandra-cluster/values.yaml

```diff
@@ -142,3 +142,6 @@ monitoring:
     # and the actual ExternalURL is still true, but the server serves requests
     # under a different route prefix. For example for use with `kubectl proxy`.
     routePrefix: ""
+
+cleaner:
+  image: burmanm/k8ssandra-client:latest
```

Comment:
We need to get this image in the k8ssandra org. It shouldn't hold up this PR though.